pkg/security/secl/model/unmarshallers.go

```diff
@@ -150,7 +151,8 @@ func (e *Process) UnmarshalBinary(data []byte) (int, error) {
 	if err != nil {
 		return 0, err
 	}
-	if IsPrintableASCII(ttyName) {
+	// naive assumption as other tty driver may create tty with other prefix
+	if IsPrintableASCII(ttyName) && (strings.HasPrefix(ttyName, "tty") || strings.HasPrefix(ttyName, "pts")) {
```

Comment:
This is great, can we maybe extract it to a separate function (like `IsPrintableASCII`)